Example/AppDelegate+CarPlay.swift

```diff
@@ -59,6 +60,23 @@ extension AppDelegate: CarPlayManagerDelegate {
         return CPListTemplate(title: "Favorites List", sections: [listSection])
     }
     
+    func carPlayManager(_ carPlayManager: CarPlayManager, leadingNavigationBarButtonsCompatibleWith traitCollection: UITraitCollection, in template: CPTemplate, for activity: CarPlayActivity) -> [CPBarButton]? {
+        
+        guard let interfaceController = self.carPlayManager.interfaceController else {
+            return nil
+        }
+        
+        switch activity {
+        case .browsing:
+            let searchTemplate = CPSearchTemplate()
+            searchTemplate.delegate = carPlaySearchController
+            let searchButton = carPlaySearchController.searchTemplateButton(searchTemplate: searchTemplate, interfaceController: interfaceController, traitCollection: traitCollection)
+            return [searchButton]
+        default:
```

Comment:
can we make this switch case exhaustive w/o using `default` to prevent regressions if we would add another activity?

Reply:
sure!